container_pipeline/workers/build.py

```diff
@@ -4,10 +4,13 @@
 import os
 import time
 
+from container_pipeline.lib import dj  # noqa
+from django.utils import timezone
+
 from container_pipeline.lib import settings
 from container_pipeline.lib.log import load_logger
 from container_pipeline.lib.openshift import Openshift, OpenshiftError
-from container_pipeline.utils import Build, get_cause_of_build
+from container_pipeline.utils import Build as BuildTracker, get_cause_of_build
```

Comment:
this could bring ambiguity, how about changing the rename of `Build` class / model itself?